python/src/aiconfig/eval/common.py

```diff
@@ -92,14 +65,14 @@ def _serialize_extra_metadata(self) -> str:
 
     name: str
     description: str
-    best_value: float
-    worst_value: float
+    best_value: MetricValue | None
+    worst_value: MetricValue | None
```

Comment:
```suggestion
    best_value: MetricValue | None = None
    worst_value: MetricValue | None = None
```
If we default to None, they don't need to be passed below, right?